README.md

```diff
@@ -28,3 +28,20 @@ fury.parse({source: '... your Swagger 2.0 document ...'}, (err, result) => {
   console.log(result.api.title);
 });
 ```
+
+### Parser Codes
+
+The following codes are used by the parser when creating warning and error annotations.
+
+Warnings:
+
+Code | Description
+---: | -----------
+   2 | Source maps are unavailable due either to the input format or an issue parsing the input.
+   3 | Data is being lost in the conversion.
+
+Errors:
+
+Code | Description
+---: | -----------
+   1 | Error parsing input (e.g. malformed YAML).
```

Comment:
I think this is more of a design comment than in context of this PR, but I'd love to see this kind of thing solved again with hypermedia. For instance, instead of having numbers that correspond to errors, we have a URL where errors are defined and we link to these errors. This is a way of decoupling from the actual implementation.

Just something to think about for the future.